src/com/itacademy/eshop/product/types/Manufacturer.java

```diff
@@ -0,0 +1,7 @@
+package com.itacademy.eshop.product.types;
+
+public enum Manufacturer {
+    MANUFACTURER1,

```

Comment:
No number in the enum value. Should be like this for this case `MANUFACTURER_ONE`